contentreview-impl/impl/src/java/org/sakaiproject/contentreview/impl/turnitin/TurnitinReviewServiceImpl.java

```diff
@@ -906,6 +926,156 @@ public void addTurnitinInstructor(Map userparams) throws SubmissionException, Tr
 	@SuppressWarnings("unchecked")
 	public void createAssignment(String siteId, String taskId, Map extraAsnnOpts) throws SubmissionException, TransientSubmissionException {
```

Comment:
Most of it belongs to the old integration, which should be deleted in the future. Maybe we can extract all that to a different method?